### DISCUSSION TITLE
How to get a Bluetooth device that is tracked into HomeAssistant

### DISCUSSION THREAD

**Discussion by stokes776** [mKeRix/room-assistant#541] [Q&A]
Can someone please point me to documenation on how to get a device to 'appear' in HomeAssistant?

Specifically a device, that is being tracked, one that I have added its bluetooth address to the config files.

Ive done so, and on the MQTT topics, I see new topics related to that MAC address, but I dont see anything new in Home Assistant about it, What am I missing here?

MQTT Topics
![image](https://user-images.githubusercontent.com/50416033/107867848-096e7f00-6e44-11eb-92a2-c533fdf8ee19.png)

On my Devices page in HA, I see a device representing the RaspberryPi doing the detecting, but nothing relating to the Bluetooth device that was being tracked.

After googling it seemed like this would be the mqtt tracking enabled on HA, but I think I've already done that (and per docs, it is on by default).
![image](https://user-images.githubusercontent.com/50416033/107867879-6ff39d00-6e44-11eb-8601-bc52b8e0a6ac.png)

What am I missing here? Do I need to manually create the thing in HA? How do I do that, it is a Device/Entity/Sensor? Im not real clear on what to even try next.

EDIT:
More info:
Adding more info: my config files.

On HomeAssistant (Raspberry Pi 4)
![image](https://user-images.githubusercontent.com/50416033/107867908-c5c84500-6e44-11eb-953b-20e1ed9dbad6.png)

On a seperate Raspberry Pi 3B+
![image](https://user-images.githubusercontent.com/50416033/107867917-db3d6f00-6e44-11eb-9401-d060500219a6.png)

**Comment by mKeRix** [comment]
Usually you configure MQTT via the integrations pane of the settings web UI in Home Assistant. Make sure that you have discovery enabled there. You may need to re-add the MQTT integration for the option to appear... not sure what the current state of affairs is for that.

**Reply by Voskov** [reply]
Do you mean like this?

```
mqtt:
  discovery: true
```

**Reply by mKeRix** [reply]
If you use the YAML config yeah, but note that I don't think you can mix YAML and UI config. If you configure the MQTT integration from the UI I think you also need to set the checkbox for MQTT discovery in the UI. (But I could be wrong, haven't tried a mixed setup.)
